src/test/resources/mockdata/compositeOrders/e5ae4afd-3fa9-494e-a972-f541df9b877e.json

```diff
@@ -87,6 +87,12 @@
       "orderFormat": "Electronic Resource",
       "owner": "LTS E-Resources & Serials",
       "paymentStatus": "Awaiting Payment",
+      "physical": {
+        "createInventory": "Instance, Holding, Item",
+        "materialSupplier": "36dd6d62-c55e-4b19-a3a0-a870da57610a",
+        "receiptDue": null,
```

Comment:
Should probably just omit null fields altogether

Reply:
updated with valid value